__std__/nix/automation/hydraJobs/default.nix

```diff
@@ -0,0 +1,4 @@
+{ inputs, cell }@block: rec
+{
+  all = import ./all.nix block;
```

Comment:
Looks like we end up with a kind of pointless `all` in the attribute names for the jobs, can we avoid that?